Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -4304,6 +4304,10 @@ static void adjustSettingsForLockdownMode(Settings& settings, const WebPreferenc
     settings.setWebAudioEnabled(false);
 #endif
     settings.setDownloadableBinaryFontAllowedTypes(DownloadableBinaryFontAllowedTypes::Restricted);
+#if ENABLE(WEB_CODECS)
+    settings.setWebCodecsEnabled(false);
+    settings.setWebCodecsAV1Enabled(false);
```

Comment:
This one is not strictly needed.

Reply:
I'll leave it as a bit of "belt-and-suspenders" feature deactivation.